Dockerfile

```diff
@@ -157,20 +158,23 @@ ENV PHYLUM_VENV="/opt/venv"
 # The following tools are susceptible to this issue and therefore have been provided
 # with explicit install/home directories that allow them to be globally accessible.
 #
+# Node and npm
+# Ref: https://github.com/tj/n
+ENV N_PREFIX="${GLOBAL_HOME}/n"
```

Comment:
This is the worst project name I've seen in a while. Good idea providing the URL to the project here, it certainly helped me.